### PR TITLE
Add check to capture stats from unbound dns resolver

### PR DESCRIPTION
**Pull request by carleton-stripe**
This check calls `sudo unbound-control stats` and parses the output.

r? @franklin-stripe @cory-stripe

### CODE REVIEW

File: conf.d/unbound.yaml.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+init_config:
+  # Not required for this check
```
Comment (franklin-stripe) [inline conf.d/unbound.yaml.example]: it might be worthwhile to add `sudo` as a flag along with a detailed comment for why it defaults to true?

Comment (carleton-stripe) [inline conf.d/unbound.yaml.example]: 👍 added that and a comment. I wasn't sure whether a URL in the comment was appropriate, but if you go to the "SETUP" portion of the man page for `unbound-control`, it describes the cert generation of the certs: https://www.unbound.net/documentation/unbound-control.html

File: checks.d/unbound.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        prefix, suffix = label.split(".", 1)
+
+        by_tag_labels = [
+            'num.query.flags',
+            'num.query.edns',
+            'num.answer.rcode',
+        ]
+
+        tags = []
+        if prefix.startswith('thread'):
```
Comment (carleton-stripe) [inline checks.d/unbound.py]: Thanks Cory! I changed the total metrics to be prefixed with total like it is in the file.  Is there a way to de-pollute datadog of the metrics I've emitted so far?

Comment (cory-stripe) [inline checks.d/unbound.py]: They will automatically roll off in a day or so. They will exist for the next 13 months (our retention) but they will no longer "pollute" the dropboxes later today!

**Comment by cory-stripe** [comment]
👍

**Comment by carleton-stripe** [comment]
r? @franklin-stripe 
Franklin, PTAL re: the sudo comment.  (I think I may have confused the ci bot w/ multiple reviewers...)

**Comment by franklin-stripe** [comment]
whoops sorry 
LGTM

**Comment by carleton-stripe** [comment]
Thanks Franklin and Cory! I'm going to mark as approved and merge since I don't know what ci bot is doing.

**Pull request merged**